src/lib/emailHelper.js

```diff
@@ -0,0 +1,6 @@
+// Return true if given email is a valid email, false either.
+module.exports.validate = function (email) {
```

Comment:
I can do that. I just copied the existing code on cozy-proxy but I can change this.